### PR TITLE
Feature/about us page

### PR DESCRIPTION
**Pull request by palmfjord**
Basically copy-pasted from https://github.com/HedvigInsurance/web/tree/master/src/sections/about-us with some sugar on top to match this repo. Sorry for the big-ass PR, but most of this code is not new so i figured its better to just push everything in one go

### CODE REVIEW

File: src/blocks/HedvigersBlock/Mobile/index.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
+import { fonts } from '@hedviginsurance/brand'
+// @ts-ignore
```
Comment (hedvigoscar) [inline src/blocks/HedvigersBlock/Mobile/index.tsx]: What's up here? 🙂

File: src/blocks/ImageBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import { colors } from '@hedviginsurance/brand/dist'
```
Comment (hedvigoscar) [inline src/blocks/ImageBlock.tsx]: 🤔

File: src/blocks/HedvigersBlock/Mobile/index.tsx (hunk 1 above)
Comment (palmfjord) [reply src/blocks/HedvigersBlock/Mobile/index.tsx]: no types and i was lazy haha. Will fix

File: src/blocks/ImageBlock.tsx (hunk 2 above)
Comment (palmfjord) [reply src/blocks/ImageBlock.tsx]: oops! I blame webstorm 😂

File: src/blocks/HedvigersBlock/Mobile/index.tsx (hunk 1 above)
Comment (palmfjord) [reply src/blocks/HedvigersBlock/Mobile/index.tsx]: actually, diving into this. Would it be _ok_ to skip? `swipeable-views` literally has 100 lines of proptypes, defining the props for this dep will be a hassle

Comment (palmfjord) [reply src/blocks/HedvigersBlock/Mobile/index.tsx]: actually, diving into this. Would it be _ok_ to skip? `swipeable-views` literally has 100 lines of proptypes, defining the props for this dep will be a hassle

Comment (hedvigoscar) [reply src/blocks/HedvigersBlock/Mobile/index.tsx]: https://www.youtube.com/watch?v=-lI3_b_bHLY

Comment (hedvigoscar) [reply src/blocks/HedvigersBlock/Mobile/index.tsx]: (go ahead and keep the ignore 😉 )

**Pull request merged**